syne_tune/optimizer/schedulers/multiobjective/utils.py

```diff
@@ -47,18 +48,32 @@ def hypervolume(
     return indicator_fn(results_array)
 
 
+def _linear_interpolate(hv_indicator: np.ndarray, indices: List[int]):
```

Comment:
What about a unit test that checks if the new linear interpolation option works correctly? 

Also more broadly, were any of the fixes in this PR for issues with the core library logic itself? If yes, those would also be good candidates for additional tests.